setup.py

```diff
@@ -10,8 +10,9 @@
       classifiers=["Programming Language :: Python :: 3 :: Only"],
       py_modules=["tap_pipedrive"],
       install_requires=[
-          "singer-python==3.6.3",
-          "requests==2.20.0",
+          "pendulum==2.0.4",
```

Comment:
@nick-mccoy pendulum is used in stream.py and tap.py, singer-python used to have it as a dependency, but it should have been listed here since the package uses it directly.

Reply:
Got it, makes sense.  Thanks!